### DISCUSSION TITLE
Issues sending messages from LHC Mobile to WhatsApp using open-wa

### DISCUSSION THREAD

**Discussion by beatriz-g-amaral** [LiveHelperChat/livehelperchat#2114] [Q&A] [answered]
Hello everyone,

I'm facing an issue with my WhatsApp integration using open-wa. Previously, I was using Telegram, and everything worked perfectly - messages sent in the chat were correctly forwarded to WhatsApp. However, I recently needed to switch to the LHC application, and now messages sent from the LHC mobile chat are not being sent to WhatsApp. The messages are sent to the chat, but they do not reach WhatsApp.

Here are more details about the issue:

I use open-wa for WhatsApp integration.
With Telegram, messages were correctly forwarded to WhatsApp.
After switching to LHC, messages sent from the LHC mobile chat are not forwarded to WhatsApp.
Has anyone else experienced this or have any ideas on what might be going wrong? I appreciate any help!

Thank you!

**Comment by remdex** [comment]
If you are talking about mobile lhc client. It can be that scenario is just not supported.

**Reply by remdex** [reply]
I'll see tomorrow if there is missing dispatch.

**Reply by remdex** [reply, marked as answer]
`chat.desktop_client_admin_msg` make soure your webhooks are listening for this event. Telegram was dispatching `chat.web_add_msg_admin` so you have to be sure there is also `chat.desktop_client_admin_msg` listener fore this. If there is no -  create identical to `chat.web_add_msg_admin`

**Reply by beatriz-g-amaral** [reply]
Adding the chat.desktop_client_admin_msg webhook solved the issue. We added the chat.desktop_client_admin_msg webhook identical to the chat.web_add_msg_admin.

**Reply by mysubcult** [reply]
> Adding the chat.desktop_client_admin_msg webhook solved the issue. We added the chat.desktop_client_admin_msg webhook identical to the chat.web_add_msg_admin.

![image](https://github.com/LiveHelperChat/livehelperchat/assets/10582537/f1124e79-93ed-4bd5-ab53-b82f9ca4aea5)
to work properly, it should look something like this
